SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/Utils/DataTypeUtilities.cpp

```diff
@@ -60,19 +55,19 @@ FString UnrealNameToSchemaComponentName(const FString& UnrealName)
 	return SchemaTypeName;
 }
 
-FString SchemaReplicatedDataName(EReplicatedPropertyGroup Group, UClass* Class, bool bPrependNamespace /*= false*/)
+FString SchemaReplicatedDataName(EReplicatedPropertyGroup Group, UClass* Class)
 {
-	return FString::Printf(TEXT("%s%s%s"), bPrependNamespace ? *GetNamespace(Class) : TEXT(""), *UnrealNameToSchemaComponentName(ClassToSchemaName[Class]), *GetReplicatedPropertyGroupName(Group));
+	return FString::Printf(TEXT("%s%s"), *UnrealNameToSchemaComponentName(ClassToSchemaName[Class]), *GetReplicatedPropertyGroupName(Group));
 }
 
-FString SchemaHandoverDataName(UClass* Class, bool bPrependNamespace /*= false*/)
+FString SchemaHandoverDataName(UClass* Class)
 {
-	return FString::Printf(TEXT("%s%sHandover"), bPrependNamespace ? *GetNamespace(Class) : TEXT(""), *UnrealNameToSchemaComponentName(ClassToSchemaName[Class]));
+	return FString::Printf(TEXT("%sHandover"), *UnrealNameToSchemaComponentName(ClassToSchemaName[Class]));
 }
 
-FString SchemaRPCComponentName(ERPCType RpcType, UClass* Class, bool bPrependNamespace /*= false*/)
+FString SchemaRPCComponentName(ERPCType RpcType, UClass* Class)
```

Comment:
I believe this one is no longer used. Can/should deprecate.

Reply:
Looks like you've addressed this in your PR.